code/builders/builder-vite/src/plugins/webpack-stats-plugin.ts

```diff
@@ -56,7 +59,9 @@ export function pluginWebpackStats({ workingDir }: WebpackStatsPluginOptions): W
     }
     // Otherwise, we need them in the format `./path/to/file.js`.
     else {
-      const relativePath = path.relative(workingDir, stripQueryParams(filename));
+      const posixWorkingDir = posix(workingDir);
```

Comment:
We're already prebundling `slash` into this package, so can we use that instead?

https://www.npmjs.com/package/slash

Reply:
@shilman great suggestion, updated